### DISCUSSION TITLE
Mate Caja thumbnails

### DISCUSSION THREAD

**Discussion by mexsudo** [SpiralLinux/SpiralLinux-project#26] [Q&A] [answered]
good morning

SpiralLinux Debian, BTFRS, SID
using Mate Desktop
mate-desktop-environment-extras metapackage

I have a problem with the Caja image thumbnails...there are none. I tried .png and .jpg 
oddly, the various video formats all have thumbnails.

PS: I also run Debian Mate and do not have this problem.
any ideas are welcome

![Screenshot at 2022-06-26 10-48-00](https://user-images.githubusercontent.com/7226539/175827269-6b2a3034-0caf-42c1-9f41-fe5965990048.png)

**Comment by geckolinux** [comment, marked as answer]
Hi there, thanks for reporting that, it looks like it's missing the `libgdk-pixbuf2.0-bin` package. It will be included by default in the next release.

**Reply by mexsudo** [reply]
excellent. I installed libgdk-pixbuf2.0-bin and all is good again.
